cloudify/plugins/workflows.py

```diff
@@ -76,6 +78,132 @@ def auto_heal_reinstall_node_subgraph(
                                        ignore_failure=ignore_failure)
 
 
+def get_groups_with_members(ctx):
```

Comment:
document what this returns, eg a docstring saying `{scaling group id: [node instance ids]}`